.github/workflows/test.yml

```diff
@@ -55,10 +55,10 @@ jobs:
           - macos-latest
           - ubuntu-latest
         version:
-          - "1.28.0"
+          - "1.32.x"
```

Comment:
`1.33.x` is better?

Reply:
Required deno version is `1.32.x`.  Hm...

Reply:
We would support the latest two last minor versions of Deno
https://github.com/vim-denops/denops.vim/pull/207#issuecomment-1317094278

I'll add above to README